server/apps/immich/src/middlewares/admin-role-guard.middleware.ts

```diff
@@ -3,36 +3,21 @@ import { Reflector } from '@nestjs/core';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
 import { UserEntity } from '@app/database/entities/user.entity';
-import { ImmichJwtService } from '../modules/immich-jwt/immich-jwt.service';
+import {ImmichJwtService} from "../modules/immich-auth/immich-jwt.service";
 
 @Injectable()
 export class AdminRolesGuard implements CanActivate {
   constructor(
-    private reflector: Reflector,
-    private jwtService: ImmichJwtService,
-    @InjectRepository(UserEntity)
-    private userRepository: Repository<UserEntity>,
-  ) {}
+      private reflector: Reflector,
+      private immichJwtService: ImmichJwtService,
+      @InjectRepository(UserEntity)
+      private userRepository: Repository<UserEntity>,
+  ) { }
```

Comment:
I think we can remove the services above

Reply:
Yes, you are right. Thanks!